.github/workflows/starknet-blocks.yml

```diff
@@ -81,12 +80,6 @@ jobs:
       - name: Install Starknet Replay deps
         run: make deps
 
-      - name: Patch Cairo Native's version
-        if: ${{ matrix.runner == 'native' }}
-        run: |
-          cd replay
-          sed -r -i 's/cairo_native.*rev.*[0-9a-z]+./cairo_native.git", rev = "${{ github.event.pull_request.head.sha }}"/' Cargo.toml
-
```

Comment:
This will only test against the default native commit on replay branch, right? We should patch de dependency to a local path, and use our current branch

Reply:
Ow, I thought is was already being done in the main branch, I'll change it

Reply:
should work now.